### PR TITLE
Ratings - Form Modal Feature

### PR DESCRIPTION
**Pull request by ryanaolea**
Used linter on ratings files for syntactical bugs. commented out a bunch of code that was causing issues on render. 
As of right now the form modal is rendering properly based on product-id in state. Features left to add for the modal are:
1. connect add photo button to photo third party api
2. connect submit button to post request
3. put a counter bellow body text input that tracks user character count as they fill in form
4. connect daniels product name to modal state for subtitle rendering correct name instead of placeholder
5. add overall rating stars for user to vote on there rating
6. connect all input from form to one massive post request to api
7. fill out a function on server for step 6

### CODE REVIEW

File: client/src/components/RatingsComponents/ReviewsList/NewReviewForm.jsx [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+  const factorGrades = {
+    'Size': ['A size too small', '1/2 size too small', 'Perfect', '1/2 size too big', 'A size too wide'],
+    'Width': ['Too narrow', 'Slightly narrow', 'Perfect', 'Slightly wide', 'Too Wide'],
+    'Comfort': ['Uncomfortable', 'Slightly uncomfortable', 'Ok', 'Comfortable', 'Perfect'],
+    'Quality': ['Poor', 'Below average', 'What I expected', 'Pretty great', 'Perfect'],
+    'Length': ['Runs short', 'Runs slightly short', 'Perfect', 'Runs slightly long', 'Runs long'],
+    'Fit': ['Runs tight', 'Runs slightly tight', 'Perfect', 'Runs slightly long', 'Runs long']
+  };
+
+  console.log(factors);
```
Comment (Daniel-Ghaly) [inline client/src/components/RatingsComponents/ReviewsList/NewReviewForm.jsx]: remove/ comment out console.log

File: client/src/components/RatingsComponents/ReviewsList/NewReviewForm.jsx [hunk 2]
```diff
@@ -0,0 +1,161 @@
...
+  // const createFactors = () => {
+
+  // }
```
Comment (Daniel-Ghaly) [inline client/src/components/RatingsComponents/ReviewsList/NewReviewForm.jsx]: Is this a new function you're working on? If not, just remove

File: client/src/components/RatingsComponents/ReviewsSidebar/AverageRating.jsx [hunk 3]
```diff
@@ -14,11 +14,12 @@ function AverageRating ({ratings}) {
...
     return roundedResult;
   }
-
+  const average = calculateRating(ratings);
   return (
     <div className='average-rating-container'>
-      <div className='numerical-rating'>{`${calculateRating(ratings)}`}</div>
-      <div className='stars-average'>Average Stars will go here</div>
+      <div className='numerical-rating'>{average}</div>
+      {/* <div className='stars-average'>Average Stars will go here</div> */}
```
Comment (Daniel-Ghaly) [inline client/src/components/RatingsComponents/ReviewsSidebar/AverageRating.jsx]: remove placeholder comment if no longer needed

**Review by Daniel-Ghaly — COMMENTED** [review]
Not much to add besides removing a few stray console.logs and removing commented out code you no longer need. Very organized and clean to read. Great job Ryan!

File: client/src/components/RatingsComponents/ReviewsList/NewReviewForm.jsx (hunk 1 above)
Comment (ryanaolea) [reply client/src/components/RatingsComponents/ReviewsList/NewReviewForm.jsx]: ✅

File: client/src/components/RatingsComponents/ReviewsList/NewReviewForm.jsx (hunk 2 above)
Comment (ryanaolea) [reply client/src/components/RatingsComponents/ReviewsList/NewReviewForm.jsx]: i might pull the functionality of this function out onto its own separate file in a refactor but for now its working and implemented in the form itself so ill just delete it for now.

**Review by Daniel-Ghaly — APPROVED** [review]

**Pull request merged**